### DISCUSSION TITLE
Import of REFI codebooks

### DISCUSSION THREAD

**Discussion by hohse** [openqda/openqda#139] [Polls]
@alessandrobelli and I tested the REFI codebook functionalities on the staging server, and found and cleaned out some bugs (okay, Alessandro did the cleaning part...).

Anyway, I made an observation regarding the import of codebooks: If you import a new codebook, it always appears to the *right* of already existing codebooks in your project (in the "Codebooks of current Project" section).
I somehow had the gut feeling or subliminal expectation that newly imported (or created) codebooks should appear to the *left* of already existing ones. I'm guessing it's because we read from left to right and at least for me it would make more sense. But I'm also aware that there are logical reasons to keep it like it is, and I might just be...weird. :)

<img width="1125" alt="Screenshot 2024-05-30 at 11 46 47" src="https://github.com/openqda/feedback/assets/9248259/c4b2f908-ca00-4b71-89f9-1077658b7b93">

So, because of this and because I wanted to try it out, I'm going to start a poll on this:

**Comment by alessandrobelli** [comment]
Related to https://github.com/openqda/openqda/pull/35 - @openqda/core-team cast your vote :)
